.github/workflows/ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "18.x"
+          node-version: "20.x"
```

Comment:
I had to upgrade the CI's node version due to this obscure bug that is hard to debug https://github.com/NomicFoundation/hardhat/issues/3877